napari/_qt/layers/qt_labels_layer.py

```diff
@@ -29,7 +29,7 @@ class QtLabelsControls(QtLayerControls):
     Attributes
     ----------
     button_group : qtpy.QtWidgets.QButtonGroup
-        Button group of labels layer modes: PAN_ZOOM, PICKER, PAINT, or FILL.
+        Button group of labels layer modes: PAN_ZOOM, PICKER, PAINT, ERASE or FILL.
```

Comment:
This line is too long... Why hasn't black caught it? :thinking: 

```suggestion
        Button group of labels layer modes: PAN_ZOOM, PICKER, PAINT, ERASE or
        FILL.
```

Reply:
not sure why black missed it/ what status of black testing doc strings is. i will fix

Reply:
just pushed change fixing doc string wrapping, making sure ERASE listed everywhere needed